esp/templates/program/modules/onsitecheckinmodule/checkin.html

```diff
@@ -49,13 +49,16 @@ <h1>Advanced Checkin &mdash; For &quot;{{module.student.name}}&quot;</h1>
 {% else %}
 <div style="margin:10px">
   <p>
-    {{module.student.name}} already checked in at <b>{{module.timeCheckedIn}}</b>.
+    {{module.student.name}} first checked in at <b>{{module.timeCheckedIn}}</b>.
```

Comment:
Instead of having "first checked in at:" and "last checked in at:" could we include info about the checkouts too? I think a nice way to have the full information would be 
"Student checked in during the following times:
(start1) to (end1)
(start 2) to present"
or something? So that we know what happened between that first and last check in? Like whether they were present at x time.

Reply:
So we want this for every single set of checkins and checkouts?

Reply:
I think so. I wouldnt expect there to be that many per student (maybe a couple per day of program) and its useful information